package/astronomer/certified/plugin.py

```diff
@@ -0,0 +1,100 @@
+from airflow.plugins_manager import AirflowPlugin
+from astronomer.certified.blueprint import ACThemeBlueprint
+import functools
+import logging
+
+from airflow.configuration import conf
+from airflow.plugins_manager import AirflowPlugin
+from airflow.utils.db import create_session
+from sqlalchemy import inspect
+
+from astronomer.version_check.update_checks import UpdateAvailableBlueprint
+
+log = logging.getLogger(__name__)
+
+update_check_interval = conf.getint("astronomer", "update_check_interval", fallback=24 * 60 * 60)
+
+
+
+
+class AstronomerCertifiedPlugin(AirflowPlugin):
+
+    name = "astronomer_certified_extensions"
+
+    flask_blueprints = [ACThemeBlueprint()]+[UpdateAvailableBlueprint()] if update_check_interval != 0 else []
```

Comment:
Slightly longer but _much_ easier to read:

```suggestion
    flask_blueprints = [ACThemeBlueprint()]
    if update_check_interval:    
        flask_blueptrints += [UpdateAvailableBlueprint()]
```